en/engine/event/event-input.md

```diff
@@ -0,0 +1,100 @@
+# Global System Events
+
+In this section, we will introduce the global system events of Cocos Creator.
+
+Global system events are irrelevant with the node hierarchy, so they are dispatched globally by `cc.systemEvent`, currently supported:
+
+- Mouse
+- Touch
+- Keyboard
+- DeviceMotion
+
+The mouse event and the touch event are similar to the node system events, except that the area of action is different. The following is a description of these events.
```

Comment:
Remove this line:
The following is a description of these events.

Reply:
I think it's important, so I've modified the statement